ztp/ztp-ran-policy-generator/sourcePolicies/MachineConfigPool.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: $name
+  labels:
+    machineconfiguration.openshift.io/role: $name
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - key: machineconfiguration.openshift.io/role
+        operator: In
+        values: [worker, $name]
```

Comment:
For the SNO use case, as broswell said , we need to point to master. We may not need to create additional mcps or new labels for nodes.

Reply:
So for du-sno, we don't really need to create MCP will just let the policies node selector is Master. Will keep this as it is for any other case.